code/API_definitions/region-device-count.yaml

```diff
@@ -635,14 +767,73 @@ components:
       content:
         application/json:
           schema:
-            $ref: "#/components/schemas/ErrorInfo"
+            allOf:
+              - $ref: "#/components/schemas/ErrorInfo"
+              - type: object
+                properties:
+                  status:
+                    enum:
+                      - 500
+                  code:
+                    enum:
+                      - INTERNAL
           examples:
             GENERIC_500_INTERNAL:
               description: Problem in Server side. Regular Server Exception
               value:
                 status: 500
                 code: INTERNAL
                 message: Unknown server error. Typically a server bug.
+    Generic501:
+      description: Not Implemented
+      headers:
+        x-correlator:
+          $ref: "#/components/headers/x-correlator"
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: "#/components/schemas/ErrorInfo"
+              - type: object
+                properties:
+                  status:
+                    enum:
+                      - 501
```

Comment:
501 MUST NOT be documented, except /count is optional to implement (which would not make sense).